tests/general/InvalidLinkBearTest.py

```diff
@@ -106,63 +101,65 @@ def assertSeverity(self, file, severity, settings={}):
                 self.assertEqual(severity_tag, out_dict['severity'])
 
     def test_run(self):
-        # Valid Links
-        valid_file = """
-        http://httpbin.org/status/200
-        http://httpbin.org/status/201
-        http://httpbin.org/status/401  # Unauthorized
-
-        # Parentheses
-        https://en.wikipedia.org/wiki/Hello_(Adele_song)/200
-
-        # Quotes
-        "https://github.com/coala/coala-bears/issues/200"
-        'http://httpbin.org/status/203'
-        ('http://httpbin.org/status/200').install_command()
-        `https://coala.io/200`
-
-        # Markup/down stuff
-        <http://httpbin.org/status/202>
-        http://httpbin.org/status/204.....
-        [httpbin](http://httpbin.org/status/200)
-        [http://httpbin.org/status/200](http://httpbin.org/status/200)
-        |http://httpbin.org/status/200|
-        <h3>Something http://httpbin.org/status/200</h3>
-        repo=\\"http://httpbin.org/status/200\\"
-
-        # Templated URLs can't be checked
-        "http://httpbin.org/{status}/404".format(...)
-        "http://httpbin.org/$status/404"
-
-        # Not a link
-        http://not a link dot com
-        http://www.%s.com
-        http://www.%d.com
-        http://www.%f.com
-
-        # Redirect
-        http://httpbin.org/status/301
-        http://httpbin.org/status/302
-
-        # Example.com URLs should be ignored
-        http://sub.example.com/404
-        """.splitlines()
-
-        self.assertResult(valid_file=valid_file)
-
-        invalid_file = """
-        http://coalaisthebest.com/
-        http://httpbin.org/status/404
-        http://httpbin.org/status/410
-        http://httpbin.org/status/500
-        http://httpbin.org/status/503
-        """.splitlines()
-
         with requests_mock.Mocker() as m:
             m.add_matcher(custom_matcher)
-            self.check_line_result_count(InvalidLinkBear(self.section,
-                                                         Queue()),
-                                         invalid_file, [1, 1, 1, 1, 1])
+
+            # Valid Links
```

Comment:
This should be moved back to be a top-level method variable.
There is no benefit in moving the variable within the `with`.
And moving it results in lots of lines being moved, which breaks the `git blame` for these lines and makes your patch 'bigger' and less obvious what you are changing.
(This applies to lots of other variables you are moving around in other test methods.)